tests/Jellyfin.XbmcMetadata.Tests/Parsers/EpisodeNfoProviderTests.cs

```diff
@@ -57,8 +104,13 @@ public void Fetch_Valid_Success()
             };
 
             _parser.Fetch(result, "Test Data/The Bone Orchard.nfo", CancellationToken.None);
-
             var item = result.Item;
+
+            // Invoke protected method to update UserData as we don't want to test the whole
+            // MetadataService here. This would need too much infrastructure setup to make sense.
+            var method = typeof(EpisodeMetadataService).GetMethod("ImportUserData", BindingFlags.Instance | BindingFlags.NonPublic);
+            method?.Invoke(_metadataService, new object[] { item, result.UserDataList, CancellationToken.None });
```

Comment:
I'm not a fan of manually invoking a protected function, but I don't know how to better do it.

Reply:
I'm not a fan of this too, but to set up the whole infrastructure to call the next public method, that calls ImportUserData was a little out of scope here ...